README.md

```diff
@@ -2,9 +2,25 @@
 
 A multi-purpose Slack bot for the MUN Computer Science Society Slack.
 
-## Setting up for development
+## Clone the repository
+`git clone https://github.com/MUNComputerScienceSociety/GarfieldBot.git`
+
+## Add your bot token
+From your Slack Workspace, find and copy your "Bot User OAuth Access Token"
```

Comment:
A suggestion should be included for the user to create their own workspace, as to not fill up ours.

Reply:
Resolved in https://github.com/SteveParson/GarfieldBot/commit/a99bec37c9ae15d8f9a738a3cd133a18f8039b3a